### PR TITLE
Prepare for re-release

### PR DESCRIPTION
**Pull request by mihhu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,104 @@
...
 ```
 
-## Samples
-Install and run the projects from the `examples` directory to see the modules in action.
+##### Properties
+###### `roomName`
+Required. String used when joining the meeting.
+
+###### `appId` 
+Optional. Provides an isolated context and prefixes the room name.
```
Comment (saghul) [inline README.md:68]: This needs to be mandatory.

File: README.md [hunk 2]
```diff
@@ -1,38 +1,104 @@
...
+
+###### `getIFrameRef`
```
Comment (saghul) [inline README.md:70]: Can you document the common properties separately, so we don't duplicate them?

**Review by saghul — CHANGES_REQUESTED** [review]
Left a few minor comments, PTAL!

File: package.json [hunk 3]
```diff
@@ -45,13 +45,12 @@
...
         "enzyme-to-json": "^3.6.2",
         "eslint": "^8.0.1",
-        "eslint-config-jitsi": "github:jitsi/eslint-config-jitsi",
         "eslint-plugin-import": "^2.25.2",
         "jest": "^27.3.1",
         "jest-cli": "^27.3.1",
         "ts-jest": "^27.0.7",
         "typescript": "^4.4.4",
-        "watch": "^1.0.2"
+        "watch": "^0.13.0"
```
[the remark this answers is not in the document]
Comment (mihhu) [reply package.json:53]: I ran npm audit fix to get rid of some vulnerabilities and the latest watch seemed to be the problem https://snyk.io/test/npm/watch/1.0.2

File: example/jitsiMeeting/src/App.jsx [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+        if (payload.isOpen || !payload.unreadCount) {
+            return;
+        }
+        apiRef.current.executeCommand('toggleChat');
+        updateLog(items => [ ...items, `you have ${payload.unreadCount} unread messages` ])
+    };
+
+    const handleKnockingParticipant = payload => {
+        /* eslint-disable-next-line */
+        debugger;
```
[the remark this answers is not in the document]
Comment (mihhu) [reply example/jitsiMeeting/src/App.jsx]: 😨 sorry

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (saghul) [reply package.json:53]: Thanks for being vigilant!

**Review by saghul — APPROVED** [review]

**Pull request merged**